src/view/shell/Composer.tsx

```diff
@@ -1,77 +1,45 @@
-import React, {useEffect} from 'react'
+import React from 'react'
+import {Modal} from 'react-native'
 import {observer} from 'mobx-react-lite'
-import {Animated, Easing, Platform, StyleSheet, View} from 'react-native'
-import {ComposePost} from '../com/composer/Composer'
+
+import {Provider as LegacyModalProvider} from '#/state/modals'
 import {useComposerState} from 'state/shell/composer'
-import {useAnimatedValue} from 'lib/hooks/useAnimatedValue'
-import {usePalette} from 'lib/hooks/usePalette'
+import {ModalsContainer as LegacyModalsContainer} from '#/view/com/modals/Modal'
+import {
+  Outlet as PortalOutlet,
+  Provider as PortalProvider,
+} from '#/components/Portal'
+import {ComposePost, useComposerCancelRef} from '../com/composer/Composer'
 
-export const Composer = observer(function ComposerImpl({
-  winHeight,
-}: {
+export const Composer = observer(function ComposerImpl({}: {
   winHeight: number
 }) {
   const state = useComposerState()
-  const pal = usePalette('default')
-  const initInterp = useAnimatedValue(0)
-
-  useEffect(() => {
-    if (state) {
-      Animated.timing(initInterp, {
-        toValue: 1,
-        duration: 300,
-        easing: Easing.out(Easing.exp),
-        useNativeDriver: true,
-      }).start()
-    } else {
-      initInterp.setValue(0)
-    }
-  }, [initInterp, state])
-  const wrapperAnimStyle = {
-    transform: [
-      {
-        translateY: initInterp.interpolate({
-          inputRange: [0, 1],
-          outputRange: [winHeight, 0],
-        }),
-      },
-    ],
-  }
```

Comment:
Oh interesting! I never looked at this code before, but I think the reason the animation didn't work is because this wasn't wrapped in `useAnimatedStyle()` maybe? Not important now. Curious that it did work in dev mode though 🤔.

Wild guess is that this gets transpiled differently based on whether it is/isn't wrapped given the babel config but I honestly have no idea.

Reply:
I think this is React Native's Animation API, not Reanimated

Reply:
Ah I missed you used `useAnimatedValue`, heh. Back to being confused why it didn't work on web lol. Guess maybe the same reason `LayoutAnimation` doesn't either...